.evergreen/config.yml

```
@@ -752,12 +752,14 @@ functions:
       working_dir: mongoc
       shell: bash
       script: |-
-        set -o errexit
         set -o errexit
         # Export the variables we need to construct URIs
         set +o xtrace
         export IAM_AUTH_ECS_ACCOUNT=${iam_auth_ecs_account}
         export IAM_AUTH_ECS_SECRET_ACCESS_KEY=${iam_auth_ecs_secret_access_key}
+        cd ../drivers-evergreen-tools/.evergreen/auth_aws/
+        . ./activate_venv.sh
+        cd -
```

Can this be simplified to the following?
```suggestion
        . ../drivers-evergreen-tools/.evergreen/auth_aws/activate_venv.sh
```

---

Good eye. Done.